nifi-stateless/nifi-stateless-core/src/main/java/org/apache/nifi/stateless/core/StatelessRemoteOutputPort.java

```diff
@@ -84,6 +87,41 @@ public StatelessRemoteOutputPort(final VersionedRemoteProcessGroup rpg, final Ve
             .build();
     }
 
+    public StatelessRemoteOutputPort(final RemoteProcessGroupDTO rpg, final RemoteProcessGroupPortDTO remotePort, final SSLContext sslContext) {
```

Comment:
There isn't much logic here but maybe the checks and defaults would warrant a common private method here as well.